### PR TITLE
Bug 1261911: remove unused/testing endpoints

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: src/api-v1.js [hunk 1]
```diff
@@ -646,48 +646,6 @@ api.declare({
 
 api.declare({
   method: 'get',
-  route: '/worker-type/:workerType/launch-specifications',
```
Comment (jhford) [inline src/api-v1.js]: Please leave this.  It's marked as experimental and I do use it.

File: src/api-v1.js [hunk 2]
```diff
@@ -779,51 +735,3 @@ api.declare({
     uptime: process.uptime(),
   });
 });
-
-api.declare({
-  method: 'get',
-  route: '/backend-status',
```
Comment (jhford) [inline src/api-v1.js]: Please do not delete this.  It is supposed to be used to get information into status.taskcluster.net.  If that's not where the data for the aws-provisioner graph is coming from, then the graph is measuring the wrong thing.

Comment (djmitche) [inline src/api-v1.js]: It's not -- status comes from the ping endpoint.  I'll file a bug to document this endpoint and its schemas, and to update the status tool (not status.tc.n) to use that endpoint

Comment (djmitche) [inline src/api-v1.js]: https://bugzilla.mozilla.org/show_bug.cgi?id=1261912

**Comment by djmitche** [comment]
ok, updated

**Pull request merged**